test/provisioner/provision_ibmcloud_initializer.go

```diff
@@ -169,6 +169,15 @@ func initProperties(properties map[string]string) error {
 	log.Infof("IksServiceURL is: %s.", IBMCloudProps.IksServiceURL)
 
 	needProvisionStr := os.Getenv("TEST_PROVISION")
+	if strings.EqualFold(needProvisionStr, "yes") || strings.EqualFold(needProvisionStr, "true") || Action == "uploadimage" {
+		if len(IBMCloudProps.ApiKey) <= 0 {
```

Comment:
As we are testing APIKEY and REGION here, we don't need them in the if block below anymore, so we should either remove them from below, or update this if check to just be `Action == "uploadimage"`. I think I slightly prefer the former, as there is less copy-paste, but I'm not too worried

Reply:
Agree, removed it from the second check